src/primitives/types/data/latch.js

```diff
@@ -0,0 +1,48 @@
+import { Parent } from "../base/parent.js";
+
+function clone(x) {
+  return x && JSON.parse(JSON.stringify(x));
+}
+
+function deepEq(a, b) {
```

Comment:
Let's replace these with the equivalent lodash.

Reply:
BTW: re `import { isEqual } from 'lodash'` vs `import isEqual from lodash.isequal`... I used to use the method-specific packages but stopped because https://lodash.com/per-method-packages

Reply:
Ah, nice - so instead I will import with a deeper relative import and we’ll be set.